src/core_ocean/shared/mpas_ocn_init_routines.F

```diff
@@ -392,7 +392,8 @@ subroutine ocn_init_routines_vert_coord(domain)!{{{
       type (block_type), pointer :: block
 
       integer :: iTracer, cell, cell1, cell2
-      real (kind=RKIND) :: normalThicknessFluxSum, thicknessSum, hEdge1, zMidPBC
+      real (kind=RKIND) :: normalThicknessFluxSum, thicknessSum, hEdge1, zMidPBC, &
+         depth1, depth2
```

Comment:
What happens if `depth1` is junk data, e.g., for `'uniform stretching'`?  Should it be set to zero below somewhere before the if statements (which could be a case construct)?

Reply:
If `depth1` is a junk value, it doesn't get used by anything.  There's no reason to set it to zero.

Reply:
These are specified with a flag now anyway.